src/sentry/lang/native/symbolicator.py

```diff
@@ -347,7 +347,10 @@ def _request(self, method, path, **kwargs):
 
         while True:
             try:
-                response = self.session.request(method, url, **kwargs)
+                with metrics.timer("events.symbolicator.request", tags={"attempt": attempts}):
+                    response = self.session.request(
+                        method, url, timeout=settings.SYMBOLICATOR_POLL_TIMEOUT + 1, **kwargs
```

Comment:
👍 Note for later: We will likely want to make this a separate setting in future.